pkg/api/handlers/compat/networks.go

```diff
@@ -285,7 +285,7 @@ func RemoveNetwork(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if !exists {
-		utils.Error(w, "network not found", http.StatusNotFound, err)
+		utils.Error(w, "network not found", http.StatusNotFound, ErrNetworkNotFound)
```

Comment:
It looks like this needs to be `config.ErrNetworkNotFound` and you need to import `"github.com/containers/libpod/pkg/network/config"`

Reply:
Yes, I'm sorry, I had forgotten the module prefix.
During installation, by the way, I had to cope with make install.tools (taken from CONTRIBUTING.md) which fails with: `install: cannot change permissions of ‘/usr/local/libexec/bats-core’: No such file or directory` since hack/install_bats.sh makes it install to /usr/local where my user account doesn't have write access to. But binaries also compile without that.